build.mill

```diff
@@ -319,6 +322,31 @@ object upickle extends Module{
         def moduleDeps = super.moduleDeps ++ Seq(ujson.native().test, core.native().test)
       }
     }
+
+    object `named-tuples` extends Module {
```

Comment:
I would assume these need some sort of "RC" version added until scala 3.7.0 is out